WordPress/gutenberg
# Migrate remaining Link UI tests to Playwright

## What?


This PR migrates all the puppeteer tests for the Link Control UI to playwright

Closes https://github.com/WordPress/gutenberg/issues/50995

## Why?

This picks up after https://github.com/WordPress/gutenberg/pull/50996 and continues with the rest of the tests. We want all our tests to be on playwright because it's easier to debug them and they are way more readable.

## How?

Following the [guidelines](https://github.com/WordPress/gutenberg/blob/971ed6c8506140421b97e38a1df77ad2666c67da/docs/contributors/code/e2e/migration.md) for e2e tests migrations

## Testing Instructions




run `npm run test:e2e:playwright -- test/e2e/specs/editor/blocks/links.spec.js` and the test should pass
